tests/app/views/test_dev_mode.py

```diff
@@ -43,3 +44,10 @@ def test_dev_flush_mode_submission(self):
                       'collection_exercise_sid': '789',
                       'ru_ref': '12346789012A'})
         self.assertStatusNotFound()
+
+    def test_extract_eq_id_and_form_type_no_form_type(self):
+        #Given
+        data_to_extract = extract_eq_id_and_form_type('rogueone.json')
+
+        self.assertEqual(data_to_extract, ('rogueone', '-1'))
```

Comment:
There could be plenty of tests involved, it depends on how many you want written? The integration tests cover a lot of the code